src/clojure_tictactoe/game/game_rules.clj

```diff
@@ -1,34 +1,31 @@
 (ns clojure-tictactoe.game.game-rules
   (:require [clojure-tictactoe.game.board :as board]))
 
+(defn horizontal-win-options [board]
+  (partition (board/side-length board) (range (count board))))
 
-(defn assess-win
-  [board]
-  (let [top-row (= (board 0) (board 1) (board 2))
-        middle-row (= (board 3) (board 4) (board 5))
-        bottom-row (= (board 6) (board 7) (board 8))
-        left-column (= (board 0) (board 3) (board 6))
-        middle-column (= (board 1) (board 4) (board 7))
-        right-column (= (board 2) (board 5) (board 8))
-        top-left-to-bottom-right-diagonal (= (board 0) (board 4) (board 8))
-        bottom-left-to-top-right-diagonal (= (board 2) (board 4) (board 6))
-        top-left-corner(board 0)
-        top-middle-space (board 1)
-        top-right-corner (board 2)
-        left-middle-space (board 3)
-        bottom-left-corner (board 6)]
-  (cond
-    top-row top-left-corner
-    middle-row left-middle-space
-    bottom-row bottom-left-corner
-    left-column top-left-corner
-    middle-column top-middle-space
-    right-column top-right-corner
-    top-left-to-bottom-right-diagonal top-left-corner
-    bottom-left-to-top-right-diagonal top-right-corner)))
-
-(defn game-over?
-  [board]
-  (not (not (or
-              (board/board-full? board)
-              (assess-win board)))))
+(defn vertical-win-options [board]
+  (apply map list (horizontal-win-options board)))
+
+(defn diagonal-win-options [board]
+  (let [side-length (board/side-length board)]
+    (let [top-left-to-bottom-right-diagonal (take side-length (range 0 (count board) (+ 1 side-length)))
+          bottom-left-to-top-right-diagonal (take side-length (range (- side-length 1) (count board) (- side-length 1)))]
+      (list top-left-to-bottom-right-diagonal bottom-left-to-top-right-diagonal))))
+
+(defn all-win-options [board]
+  (concat (diagonal-win-options board) (vertical-win-options board) (horizontal-win-options board)))
+
+(defn potential-wins [board]
```

Comment:
What does this function do?

Reply:
Nah, I'm highlighting the `potential-wins` function. Not `all-win-options`

Reply:
Ah! Okay, so `potential-wins` kind of "fills in" the options of `all-win-options`. It's the step in between "here are all the ways to win" and "did someone win that way?". Also open to better naming options for that one.

Reply:
🤔 I would probably suggest just condensing them into one function (potential-wins and all-win-options). And just having that function tell you what you need to know.

Reply:
Cool. I'm going to leave it as is for now, but definitely have this in the back of my mind as I go forward.